frontend/src/views/TE/Task.vue

```diff
@@ -187,6 +192,7 @@ export default class Task extends Mixins<TaskMixin, FormatTime>(
   task: any = {};
   isFetchingTask = true;
   isCoordinator = false;
+  fetchTaskError = false;
```

Comment:
Please read https://itnext.io/managing-local-state-in-vue-js-components-d5d92fb432 and decide which of two approaches is suitable for this component. I opt for “String” approach as the states are mutually exclusive.